src/components/protocol_handler/test/protocol_packet_test.cc

```diff
@@ -49,26 +73,39 @@ class ProtocolPacketTest : public ::testing::Test {
     some_session_id = 0xFEDCBA0;
     some_connection_id = 10;
   }
+
+  RawMessagePtr GetRawMessage(uint8_t version,
+                              uint8_t frame_type,
+                              uint8_t service_type) {
+    ProtocolPacket prot_packet(some_connection_id,
+                               version,
+                               PROTECTION_OFF,
+                               frame_type,
+                               service_type,
+                               FRAME_DATA_HEART_BEAT,
+                               some_session_id,
+                               0u,
+                               some_message_id);
+    EXPECT_EQ(frame_type, prot_packet.frame_type());
+    return prot_packet.serializePacket();
+  }
+
   uint32_t some_message_id;
   uint32_t some_session_id;
   ConnectionID some_connection_id;
 };
 
```

Comment:
@AByzhynar google style: Data members of classes, both static and non-static, are named like ordinary nonmember variables, but with a trailing underscore.